web-console/src/lib/components/streaming/import/randomData/generators.ts

```diff
@@ -191,12 +190,28 @@ export const columnTypeToRngOptions = (type: ColumnType): IRngGenMethod[] => {
         assert(type.component !== undefined && type.component !== null)
         return transformToArrayGenerator(type, columnTypeToRngOptions(type.component))
       })
-      .otherwise(() => {
-        throw new Error('unsupported type for RNG')
-      })
+      .otherwise(() => UNSUPPORTED_TYPE_GENERATORS)
   )
 }
 
+const UNSUPPORTED_TYPE_GENERATORS: IRngGenMethod[] = [
+  {
+    title: 'Constant',
+    category: Categories.DEFAULT,
+    generator: () => '',
```

Comment:
```suggestion
    generator: (ct, settings) => settings?.value ?? '',
```

this might be better as it will use the value from the TextField declared in form_fields below (I think, you'll have to test it).. Sorry this didn't occur to me yesterday.